tiltak/src/main/kotlin/no/nav/amt/tiltak/tiltak/controllers/TiltakInstansController.kt

```diff
@@ -1,44 +1,42 @@
 package no.nav.amt.tiltak.tiltak.controllers
 
 import no.nav.amt.tiltak.core.domain.tiltak.Deltaker
-import no.nav.amt.tiltak.core.domain.tiltak.TiltakInstans
+import no.nav.amt.tiltak.core.port.TiltakService
 import no.nav.amt.tiltak.tiltak.controllers.dto.TiltakDeltagerDto
-import no.nav.amt.tiltak.tiltak.controllers.dto.TiltakDto
 import no.nav.amt.tiltak.tiltak.controllers.dto.TiltakInstansDto
+import no.nav.amt.tiltak.tiltak.controllers.dto.toDto
 import no.nav.security.token.support.core.api.Protected
+import org.springframework.http.HttpStatus
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.server.ResponseStatusException
 import java.time.LocalDate
 import java.time.ZonedDateTime
 import java.util.*
 
 @RestController
 @RequestMapping("/api/tiltak-instans")
-class TiltakInstansController {
+class TiltakInstansController(
+	private val tiltakService: TiltakService,
+) {
 
 	@Protected
 	@GetMapping("/{tiltakInstansId}")
 	fun hentTiltakInstans(@PathVariable("tiltakInstansId") tiltakInstansId: String): TiltakInstansDto {
-		return TiltakInstansDto(
-			id = UUID.randomUUID(),
-			navn = "Truckførerkurs",
-			startdato = ZonedDateTime.now().plusDays(1),
-			sluttdato = ZonedDateTime.now().plusDays(3),
-			status = TiltakInstans.Status.IKKE_STARTET,
-			deltagerAntall = 8,
-			deltagerKapasitet = 4,
-			TiltakDto(
-				tiltakskode = "GRUPPEAMO",
-				tiltaksnavn = "Gruppe AMO"
-			),
-		)
+		val instansId = UUID.fromString(tiltakInstansId)
+		val instans = tiltakService.getTiltakInstans(instansId)?: throw ResponseStatusException(HttpStatus.NOT_FOUND, "Fant ikke tiltakinstans")
+		val tiltak = tiltakService.getTiltak(instans.tiltakId)?: throw ResponseStatusException(HttpStatus.INTERNAL_SERVER_ERROR, "Fant ikke tiltak")
+		//Hvordan sjekke om innlogget bruker har tilgang til tiltaksgjennomføring?
+
+		return instans.toDto(tiltak);
+
 	}
 
 	@Protected
-	@GetMapping("/{tiltkInstansId}/deltagere")
-	fun hentDeltagere(@PathVariable("tiltkInstansId") tiltkInstansId: String): List<TiltakDeltagerDto> {
+	@GetMapping("/{tiltakInstansId}/deltagere")
```

Comment:
Litt forskjellig bruk av "Deltager" og "Deltaker", vi bør bestemme oss for en

Reply:
Trodde vi hadde bestemt oss for deltager men ser nå at det er litt forskjellig. Det er forsåvidt andre ting vi har blitt inkonsekvente med på navngivning som Tiltaksinstans/TiltakInstans, som gjør det vanskeligere å finne frem. Tenker jeg kan lage en ny pr på det så alt blir likt